crates/core/tedge/src/cli/connect/bridge_config.rs

```diff
@@ -40,6 +40,7 @@ impl BridgeConfig {
         writeln!(writer, "address {}", self.address)?;
 
         // XXX: This has to go away
+        dbg!(&self.bridge_root_cert_path);
```

Comment:
If this checkin wasn't accidental, you might wanna add more context to what's being printed here, so that someone reading the debug log knows what this value is.

Reply:
removed the debug statement.